distribution/dashboard/ui/src/services/tenant.ts

```diff
@@ -18,35 +18,41 @@ export async function getTenant({
 }: API.NamespaceAndName): Promise<API.TenantBasicInfoResponse> {
   let r = await request(`${tenantPrefix}/${ns}/${name}`, {
     method: 'GET',
-  })
-  let res:API.TenantBasicInfo = {
-    info:{
-      charset:'',
-      clusterName:'',
-      tenantName:'',
-      tenantRole:'',
-      unitNumber:0,
-      status:'',
-      name:'',
-      namespace:'',
-      locality:''
+  });
+  let res: API.TenantBasicInfo = {
```

Comment:
Similarly to the following, I actually don't quite understand why objects are used here, nor is it necessary to set default values because they will be overwritten by forEach below. You can take a look at how to design them better

Reply:
I need to use Object.keys to access the attributes of the info object.